ui/tests/acceptance/client-history-test.js

```diff
@@ -0,0 +1,282 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click, settled } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import Pretender from 'pretender';
+import authPage from 'vault/tests/pages/auth';
+import { addMonths, format, formatRFC3339, startOfMonth, subMonths } from 'date-fns';
+import { create } from 'ember-cli-page-object';
+import { clickTrigger } from 'ember-power-select/test-support/helpers';
+import ss from 'vault/tests/pages/components/search-select';
+import {
+  generateActivityResponse,
+  generateConfigResponse,
+  generateLicenseResponse,
+  SELECTORS,
+  sendResponse,
+} from '../helpers/clients';
+
+const searchSelect = create(ss);
+
+module('Acceptance | clients history tab', function (hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(function () {
+    return authPage.login();
+  });
+
+  hooks.afterEach(function () {
+    this.server.shutdown();
+  });
+
+  test('shows warning when config off, no data, queries available', async function (assert) {
+    const licenseStart = startOfMonth(subMonths(new Date(), 6));
+    const licenseEnd = addMonths(new Date(), 6);
+    const license = generateLicenseResponse(licenseStart, licenseEnd);
+    const config = generateConfigResponse({ enabled: 'default-disable' });
+    // const activity = generateActivityResponse(0, licenseStart, licenseEnd);
+    this.server = new Pretender(function () {
+      this.get('/v1/sys/license/status', () => sendResponse(license));
+      this.get('/v1/sys/internal/counters/activity', () => sendResponse(null, 204));
+      this.get('/v1/sys/internal/counters/config', () => sendResponse(config));
+      this.get('/v1/sys/version-history', () => sendResponse({ keys: [] }));
+      this.get('/v1/sys/health', this.passthrough);
+      this.get('/v1/sys/seal-status', this.passthrough);
+      this.post('/v1/sys/capabilities-self', this.passthrough);
+      this.get('/v1/sys/feature-flags', this.passthrough);
+    });
+    await visit('/vault/clients/history');
+    assert.equal(currentURL(), '/vault/clients/history');
+    assert.dom(SELECTORS.activeTab).hasText('History', 'history tab is active');
+
+    assert.dom('[data-test-tracking-disabled] .message-title').hasText('Tracking is disabled');
+    assert.dom(SELECTORS.emptyStateTitle).hasText('No data received');
+    assert.dom(SELECTORS.filterBar).exists('Shows filter bar to search previous dates');
+    assert.dom(SELECTORS.usageStats).doesNotExist('No usage stats');
+    // TODO: has no end filter date
```

Comment:
Confirmed with Ivana - and we should only show the calendar widget if there is something to filter - so in this case the user should have to input an older start date that returns data before seeing the filter bar.